redbot/cogs/trivia/data/lists/doom.yaml

```diff
@@ -0,0 +1,2230 @@
+- E3M3
+- Pandemonium
+- "E4M3: Sever the Wicked"
+- E4M3
+- Sever the Wicked
+"Which difficulty level is shortened to ITYTD?":
+- I'm Too Young To Die
+- Im Too Young To Die
+"Which difficulty level is shortened to HNTR?":
+- Hey, Not Too Rough
+- Hey Not Too Rough
+"Which difficulty level is shortened to HMP?":
+- Hurt Me Plenty
+"Which difficulty level is shortened to UV?":
+- Ultra Violence
+- Ultra-Violence
+- Ultraviolence
+"Which difficulty level is shortened to NM?":
+- Nightmare!
+- Nightmare
+"Which powerup gives Doomguy +100 health?":
+- Supercharge
+- Soulsphere
+"Which Classic Doom powerup renders player's view in inverted grayscale?":
+- Invulnerability
+- Invincibility
+"Which Classic Doom powerup renders all sectors with full brightness?":
+- Light amplification visor
+- Lite-amp
+- Liteamp
+- Goggles
+"Which Classic Doom powerup increases strength of the melee attack tenfold?":
+- Berserk
+"Which Classic Doom powerup reveals the level's architecture?":
+- Computer area map
+- Computer map
+- Full map
+- Automap
+"Which Classic Doom powerup's protective effect randomly fails?":
+- Radiation shielding suit
+- Radiation suit
+- Environmental suit
+- Hazard suit
+- Protective suit
+- Protection suit
+"Which powerup renders Doomguy in the same way as the Spectre monster?":
+- Partial invisibility
+- Blursphere
+- Blur
+"What does ''BFG'' stand for?":
+- Big Fucking Gun
+"What does ''UAC'' stand for?":
+- Union Aerospace Corporation
+"In which Doom installment was Deathmatch featured first?":
+- Doom 1
+- 1
+- Original
+- Initial
+"Which weapon is vital to completing Map30: Icon of Sin?":
+- Rocket Launcher
+- Rocket
+- RL
+"Which Classic Doom monster has been officially sold as a marketable plushie?":
+- Cacodemon
+- Caco
+- Tomato
+- Raspberry
+"What pet does Doomguy have?":
+- Rabbit
+"What is the name of Doomguy's pet?":
+- Daisy
+"Which country took issue with the Easter egg secret levels in Doom 2?":
+- Germany
+- Deutschland
+- de
+"Enter any Classic Doom cheat code.":
+- idbehold
+- idchoppers
+- idclip
+- idclev
+- iddqd
+- iddt
+- idfa
+- idkfa
+- idmus
+- idmypos
+- idspispopd
+"What is the two letter prefix of all of the Classic Doom cheat codes?":
+- id
+"Which Classic Doom weapon can be specifically obtained with a cheat?":
+- Chainsaw
+"Which Classic Doom weapon is accompanied by a pickup message \"Find some meat!\"?":
+- Chainsaw
+"How many distinct keys are there in Classic Doom?":
+- Six
+- 6
+"Name any key from Classic Doom.":
+- Blue keycard
+- Yellow keycard
+- Red keycard
+- Blue key card
+- Yellow key card
+- Red key card
+- Blue card
+- Yellow card
+- Red card
+- Blue skullkey
+- Yellow skullkey
+- Red skullkey
+- Blue skull key
+- Yellow skull key
+- Red skull key
+- Blue skull
+- Yellow skull
+- Red skull
+"In which Doom installment was Chainsaw featured first?":
+- Doom 1
+- 1
+- Original
+- Initial
+"In which Doom installment was Super Shotgun featured first?":
+- Doom 2: Hell on Earth
```

Comment:
```suggestion
- "Doom 2: Hell on Earth"
```
You remembered quotes for most of them, but this one got overlooked, and is making the checks fail currently.